tutorials/algorithms/04_vqd.ipynb

```diff
@@ -0,0 +1,292 @@
+      "text/plain": [
+       "<Figure size 507.997x204.68 with 1 Axes>"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from qiskit.circuit.library import TwoLocal\n",
+    "from qiskit.algorithms.optimizers import SLSQP\n",
+    "\n",
+    "num_qubits = 2\n",
+    "ansatz = TwoLocal(3, rotation_blocks=[\"ry\", \"rz\"], entanglement_blocks=\"cz\", reps=1)\n",
+    "\n",
+    "optimizer = SLSQP()\n",
+    "ansatz.draw('mpl')"
```

Comment:
It might be nicer to use `ansatz.decompose().draw()` so a user can see what the circuit actually looks like instead of a single block.